test/Unit/Protocol/Smtp/ReplyTest.php

```diff
@@ -109,4 +109,29 @@ public function it_an_error_500()
         $this->assertTrue($reply->isError());
     }
 
-}
\ No newline at end of file
+    /**
+     * @test
+     */
+    public function it_an_implemented_command()
```

Comment:
it_**is**_an

Reply:
Both updated, plus the ones above that I copied :wink: